explainer.md

```diff
@@ -532,6 +532,23 @@ function drawScene() {
 }
 ```
 
+### Changing the Field of View for inline sessions
+
+Whenever possible projection matrices delivered by the API should make use of physical properties such as the headset optics or camera lens to determine the field of view to use. However, for most inline content there aren't any physically based values to draw from. In order to provide a unified render pipeline for inline content an arbitrary field of view must be selected.
```

Comment:
nit: "to draw from" is a bit too close to sounding like a double entendre.  Perhaps "from which to infer a field of view".